orttraining/orttraining/python/training/optim/_ds_modifier.py

```diff
@@ -10,10 +10,12 @@
 # - has_overflow_partitioned_grads_serial : https://github.com/microsoft/DeepSpeed/blob/d8e9ef6f99e27bb95e10bd146d145b3372b4cfda/deepspeed/runtime/zero/stage2.py#L1799
 # --------------------------------------------------------------------------
 
-import torch
 import types
 import warnings
 from distutils.version import LooseVersion
+
+import deepspeed
```

Comment:
we cannot put "import deepspeed" here, it might break when import the module without deepspeed installed.

Reply:
right. just fixed it.